DeepMRSeg/deepmrseg_test.py

```diff
@@ -38,63 +43,57 @@ def read_flags():
 	# Only parse for "--config"
 	configarg,_ = parser.parse_known_args()
 
-#	INPUT LIST
+#	I/O
 #	==========
-	inputArgs = parser.add_argument_group( 'REQUIRED INPUT ARGS' )
+	inputArgs = parser.add_argument_group( 'INPUT SCANS - REQUIRED', 'Select one of the 3 options: Single case (inImg/outImg), process all scans in a folder (inDir/outDir/outSuff) or using a scan list (sList)')
+	
+	## I/O Option1: Single scan I/O
+	inputArgs.add_argument( "--inImg", action='append', default=None, type=str, \
```

Comment:
We need to add some more info here for clarification. Should multiple input images to comma separated or multiple arguments?